_translations/index_ta.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: signed
+title: ரிச்சர்ட் ஸ்டால்மனுக்கு ஆதரவு கடிதம்
+description: ரிச்சர்ட் ஸ்டால்மன் FSF சேர்வதற்கு ஆதரவு கடிதம்
+image: /assets/social-media-preview.png
+locale: ta_IN
+twitter:
+  card: summary_large_image
+---
+
+2021-03-23
+
+ரிச்சர்ட் ஸ்டால்மன் என்பவர் சாஃப்ட்வேர் சுதந்திரத்திற்காக பலவருடம் போராடினார். அவர் GNU operating system மற்றும் EMACS சாப்ட்வேரை உருவாக்கினார்.
+
+கடந்த சில மாதங்களாக, அவர் தனது கருத்துக்களை பதிவு செய்ததற்காக அவரை நீக்க கோரி இணையதளங்களில் பலரும் போராடிக் கொண்டிருக்கின்றனர். இதேபோல் பல நபர்களை அவர்கள் தாக்கினர். ஆனால் இம்முறை நாம் அமைதியாக இருக்க மாட்டோம்.
+
+FSF அவர்கள் தனியாக முடிவு செய்யூம் தகுதியுள்ளவர்கள். அதனால் அவர்களின் தொண்டர்களை நீக்கவோ சேர்க்க உரிமை அவர்களுக்கு இருக்கிறது. 
+மட்டும் வெளிய அழுத்தத்துக்கு அவர்கள் கட்டுப்படக் கூடாது. ரிச்சர்ட் ஸ்டால்மன் அவர் தொடர்பான கருத்துக்களை அவர்கள் ஆழ்ந்து பரிசீலிக்க வேண்டும்.
+
+கடந்த காலங்களை பார்க்கையில், ரிச்சர்ட் ஸ்டால்மனின் கருத்துக்கள் பலரையும் அசவுகரியம் உண்டாக்கியுள்ளது. பொதுவாக அவர் சுதந்திரத்திற்காகப் போராடுவது மற்றும் சில நேரம் மக்களின் கருத்துக்களுக்கு மதிப்பு அதிகமாகக் கொடுக்க மாட்டார். ஆதலால் அவர் எந்த கருத்து சொன்னாலும் அவரை மக்கள் தாக்குகின்றனர் அவரைப் புரிந்து கொள்ளமாட்டார்கள்.
+
+என்ன இருந்தாலும் அவருக்கு அவர் கருத்து சொல்ல உரிமை இருக்கின்றது. அவரின் கருத்துக்கும் அவரு சமூகத்தை வழிநடத்தும் திறமைக்கும் சம்பந்தமில்லை. ஆதரவாளர்கள் அவர் சொல்வதற்கு ஒத்துக் கொள்ளத் தேவை இல்லை ஆனால் நம்மைப்போல் அவர் என்ன வேண்டுமென்றாலும் சொல்லலாம்.
+
+**FSF குழுவே:**
+
+அவரை நீக்குவது உங்களது இயக்கத்தை பெரிதும் பாதிக்கும். ஆதலால் நீங்கள் எடுக்கும் முடிவுகளை ஆழ்ந்து பரிசீலிக்குமாறு நாங்கள் கேட்டுக் கொள்கிறோம். உங்கள் முடிவு சாப்ட்வேர் உலகையே பாதிக்கும். சிந்தியுங்கள்.
+    
+**ரிச்சர்ட் ஸ்டால்மனின் கருத்துக்களை கூறியதற்காக அவரைத் தாக்க காத்திருக்கும் ரவுடி கும்பலுக்கு:**
+
+எந்த ஒரு சமூகத்திற்கும் தலைவரை தேர்ந்தெடுக்கும் உரிமை உங்களுக்கு இல்லை. ரவுடி கும்பல் மூலம் அவரின் நீக்க நினைக்காதீர். விவாதத்திற்கு வாரீர்!
+
+To sign, please [submit a pull
+request](https://github.com/rms-support-letter/rms-support-letter.github.io/pulls).
+To sign without using Github, you can:
+- Send a signed patch to [signrms@prog.cf](mailto:signrms@prog.cf) 
+- Or to [~tyil/rms-support@lists.sr.ht](mailto:~tyil/rms-support@lists.sr.ht).
+- Or add your signature here: [https://codeberg.org/rms-support-letter/rms-support-letter/issues/1](https://codeberg.org/rms-support-letter/rms-support-letter/issues/1)
```

Comment:
It'd probably be better if you translate this as well.

Reply:
It's damn hard to translate those. Let it be